src/main/java/duke/ui/MainWindow.java

```diff
@@ -0,0 +1,59 @@
+package duke.ui;
+
+import duke.Duke;
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
+/**
```

Comment:
Maybe add a blank line before line 11 for better consistency / readability.

Reply:
Thanks for pointing that out!